src/settings/FileExtensions/resultsFormatter.js

```diff
@@ -0,0 +1,51 @@
+import React, { Fragment } from 'react';
+import { FormattedDate } from 'react-intl';
+
+import { Icon } from '@folio/stripes/components';
+
+import css from './FileExtensions.css';
+
+const resultsFormatter = intl => ({
+  importBlocked: record => {
```

Comment:
Consider using methods syntax in this object
```suggestion
  importBlocked(record) {
```

Reply:
I will keep the way it is now for consistency.